assets/js/ga.js

```diff
@@ -0,0 +1,6 @@
+(function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
```

Comment:
As much as it pains me, we should be wrapping this in Do Not Track, shouldn't we @cadecairos?

Reply:
Yeah totally, we can probably just file it.